core/admin/mailu/configuration.py

```diff
@@ -87,8 +87,7 @@
     'HOST_REDIS': 'redis',
     'HOST_FRONT': 'front',
     'SUBNET': '192.168.203.0/24',
-    'SUBNET6': None,
```

Comment:
hmm, SUBNET6 is still used; this must be a typo

Reply:
I removed the comma after `SUBNET6`, right?

Reply:
doh, sorry. I need new specs!